Source/SourceKittenFramework/library_wrapper.swift

```diff
@@ -1,16 +1,33 @@
 import Foundation
+#if os(Windows)
+import WinSDK
+#endif
 
 // MARK: - Shared Types & Functions
 
 struct DynamicLinkLibrary {
-    fileprivate let handle: UnsafeMutableRawPointer
+#if os(Windows)
+    typealias HandleType = HMODULE?
+#else
+    typealias HandleType = UnsafeMutableRawPointer
+#endif
```

Comment:
Nit: Generally typealiases in Swift aren't suffixed with `Type`

```suggestion
#if os(Windows)
    typealias Handle = HMODULE?
#else
    typealias Handle = UnsafeMutableRawPointer
#endif
```